packages/teams-js/src/public/dialog.ts

```diff
@@ -16,21 +17,44 @@ import { runtime } from './runtime';
  *
  * @beta
  */
+export interface SdkResponse {
+  err: string;
+  result: string | object;
+}
+
 export namespace dialog {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  export type PostMessageChannel = (message: any, onComplete?: (status: boolean, reason?: string) => void) => void;
+  export type DialogSubmitHandler = (result: SdkResponse) => void;
   /**
    * Allows an app to open the dialog module.
    *
-   * @param dialogInfo - An object containing the parameters of the dialog module
+   * @param urlDialogInfo - An object containing the parameters of the dialog module
    * @param submitHandler - Handler to call when the task module is completed
```

Comment:
This comment should probably not say task module anymore. Maybe something like "This handler is called when the dialog has been submitted or closed"

Reply:
Will change!